docs/backstage.md

```diff
@@ -0,0 +1,124 @@
+
+If you want to make changes to this image such as adding a different domain or new software catalogs you will need to make your changes, build your own image and change the deployment manifest to reference the image you have created. The source code for Backstage is found in the root Backstage folder. To build the image follow the steps below:
+
+1. **Fork & Clone the Repository**:
+    - First, fork the repository to your own GitHub account by clicking the "Fork" button on the repository page.
+    - Then, clone your forked repository:
+    ```sh
+    git clone https://github.com/<your_fork>/aks-platform-engineering.git
+    cd aks-platform-engineering
+
+2. **Install Dependencies**: Ensure all dependencies are installed by running:
+
+    ```sh
+    yarn install
+    ```
+
+3. **Build the Project**: Run the build script defined in your `package.json`. Based on your previous commands, it looks like you need to build the backend:
+
+    ```sh
+    yarn build:backend --config ./app-config-local.yaml
+    ```
+
+4. **Optional - Run the Application Locally**: After building the project, you can run it locally. Ensure that all necessary environment variables are set:
+
+    ```sh
+    export BASE_URL=https://your-local-base-url.com
+    export POSTGRES_HOST=your-local-postgres-host
+    export POSTGRES_PORT=your-local-postgres-port
+    export POSTGRES_USER=your-local-postgres-user
+    export POSTGRES_PASSWORD=your-local-postgres-password
+    export POSTGRES_DB=your-local-postgres-db
+
+    yarn start
+    ```
+
+5. **Ensure Azure CLI is Installed**: Make sure you have the Azure CLI installed and logged in. If not, install it from [here](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) and log in using:
+
+    ```sh
+    az login
+    ```
+
+6. **Set Environment Variables**: Set the `ACR_NAME` and `RESOURCE_GROUP` environment variables:
+
+    ```sh
+    export ACR_NAME=your_acr_name
+    export RESOURCE_GROUP=your_resource_group
+    ```
+
+7. **Build the Docker Image Locally**: Use the `docker build` command to build your Docker image:
+
+    ```sh
+    docker build -t $ACR_NAME.azurecr.io/my-backend-app:latest .
+    ```
+
+8. **Login to Azure Container Registry**: Use the Azure CLI to log in to your ACR:
+
+    ```sh
+    az acr login --name $ACR_NAME --resource-group $RESOURCE_GROUP
+    ```
+
+9. **Push the Docker Image to ACR**: Push the built image to your ACR:
+
+    ```sh
+    docker push $ACR_NAME.azurecr.io/my-backend-app:latest
+    ```
+
+10. **Verify the Image in ACR**: You can verify that the image has been pushed to ACR by listing the repositories:
+
+    ```sh
+    az acr repository list --name $ACR_NAME --output table
+    ```
+
+
+## Getting Started
```

Comment:
Let's move this section up top then people can decide to build their own image after if they want.